### DISCUSSION TITLE
Image Upload Failing

### DISCUSSION THREAD

**Discussion by steffanmouton** [area17/twill#1315] [Q&A]
Hi there!
I've got a fresh install of Twill 2.5 onto an existing Laravel project. I am following the youtube setup video which worked flawlessly up until I actually uploaded an image. It is giving me an error "Response does not include the base64 encoded policy!"

![image](https://user-images.githubusercontent.com/36455372/146395379-b71ef43f-da2e-4ca9-a0aa-f89bbf4490b4.png)

I've done some googling and tried resetting the caches and used different browsers. I'm at a loss now.

My .env file has the following:
`MEDIA_LIBRARY_ENDPOINT=local`

`MEDIA_LIBRARY_LOCAL_PATH=uploads/`

`MEDIA_LIBRARY_IMAGE_SERVICE=A17\Twill\Services\MediaLibrary\Glide`

`MEDIA_LIBRARY_FILESIZE_LIMIT=50`

Any assistance would be wonderful! 
Thanks!
Stef

**Comment by pboivin** [comment]
Hi @steffanmouton, your `.env` configuration looks good to me. Do you get any other error in the browser's console?

**Reply by steffanmouton** [reply]
I'm not sure why I didn't think to check for that...
Looks like maybe it's attempting to reach into s3 for some reason? I believe I've got it all set up for local development. 
![image](https://user-images.githubusercontent.com/36455372/146457935-978b2567-ca96-48fb-8395-3e5533ba27c5.png)
![image](https://user-images.githubusercontent.com/36455372/146458010-afa213aa-3740-4402-80b8-11d9b90f21c2.png)

Any ideas why it would be reaching out to s3 or aws if I have the Twill env set to local? Could another part of the code base be attempting to intercept it?

Our client has tasked us with implementing a cms on top of their existing, mostly simple, code base. There was no CMS previously, they were manually adding items to the database.

**Comment by pboivin** [comment]
This will change in the next major release but for now, Twill config is set to S3 out of the box.

Is it possible that you have cached configuration? You mentioned clearing the caches, but have you tried: `php artisan config:clear`?

**Comment by steffanmouton** [comment]
I ran that and am getting the same error.
